IFC4x3/PropertyConstants/i/INTERNALSTAFF_23JTDKB_5CNBaN6AkA1eoQ.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="INTERNALSTAFF_23JTDKB_5CNBaN6AkA1eoQ" Name="INTERNALSTAFF" UniqueId="834dd354-2fe1-4c5c-b917-18ab8a068c9a" />
+
```

Comment:
Documentation missing. See also other comment regarding ConstructionCompany.

Reply:
adb5fdd751ea322bf0649365b080a81b2a458a62

Reply:
This enum is still here,

Reply:
Done. Here